q2_types/feature_data/_transformer.py

```diff
@@ -191,6 +191,8 @@ def _23(ff: TSVTaxonomyFormat) -> pd.Series:
 @plugin.register_transformer
 def _29(ff: TSVTaxonomyFormat) -> qiime2.Metadata:
     df = _taxonomy_formats_to_dataframe(str(ff), has_header=True)
+    for index in range(len(df['Taxon'])):
```

Comment:
I just googled and learned that pandas has a [built-in method](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.Series.str.strip.html) for this!

```python
df['Taxon'] = df['Taxon'].str.strip()
```

No need to loop over all the records, let pandas do it for you! :tada:

Reply:
Will also need to do the same up in https://github.com/qiime2/q2-types/pull/219/files#diff-2ff029ab1eb8bde1e71891045ee32b37R169, as per the [original issue](https://github.com/qiime2/q2-types/issues/179#issuecomment-439436700).

Reply:
Sorry, I am just seeing the comments on this PR now. I have implemented the change you recommended for methods that start on lines 169 and 194 respectively. :grin:

Reply:
Thanks @David-Rod - have you pushed those changes up yet? The PR doesn't appear to be updated.